bindings/iOS/all.h

```diff
@@ -184,6 +190,7 @@
 #import <VerovioFramework/positioninterface.h>
 #import <VerovioFramework/preparedatafunctor.h>
 #import <VerovioFramework/proport.h>
+#import <VerovioFramework/quilisma.h>
```

Comment:
Just checking that these are supposed to be here? They don't seem like they're immediately relevant to this PR?

Reply:
It happens for the `iOS` framework to be outdated, and it was the case here. In any case, it is always updated during releases, but I am also trying to keep it updated whenever new files are added.

Reply:
OK!